apps/meteor/client/sidebar/header/actions/Administration.tsx

```diff
@@ -46,13 +47,12 @@ const Administration: VFC<Omit<HTMLAttributes<HTMLElement>, 'is'>> = (props) =>
 	const hasAuditLicense = useHasLicenseModule(AUDIT_LICENSE_MODULE) === true;
 	const hasAuditPermission = useAtLeastOnePermission(AUDIT_PERMISSIONS) && hasAuditLicense;
 	const hasAuditLogPermission = useAtLeastOnePermission(AUDIT_LOG_PERMISSIONS) && hasAuditLicense;
-	const hasManageApps = useAtLeastOnePermission(MANAGE_APPS_PERMISSIONS);
 	const hasAdminPermission = useAtLeastOnePermission(ADMIN_PERMISSIONS);
-	const showMenu = hasAuditPermission || hasAuditLogPermission || hasManageApps || hasAdminPermission || !!accountBoxItems.length;
+	const hasManageApps = useAtLeastOnePermission(MANAGE_APPS_PERMISSIONS);
```

Comment:
useAtLeastOnePermission at least implies more than one

Reply:
since the menu is going to be displayed forever the logic/permission can be moved to inside